packages/jaeger-ui/src/components/TracePage/CriticalPath/testResults/test1.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+
+  │                            span A                                   │
+  ++++┌────────────────────────────────────────────────────────┐++++++++++
+      │                         span B                         │
+      +++++++++++++++++++++++++++++++──────────────────────+++++
+                                    │      span C          │
+                                    +++──────+++++──────┐+++
+                                      │span D│   │span E│
+                                      ++++++++   ++++++++
+
+Here +++++ are critical path sections
+
+*/
+const startTime = 1579070675083000;
+
+const criticalPathSections = [
```

Comment:
A nitpicking suggestion 😛, but you can use some human readable numbers for the timestamp; at the moment, we have some unix numbers, which are not easily understandable. You can use some small numbers for defining a span, for e.g. `SpanB{start_time=startTime + 10, end_time=startTime + 110, spanID="SpanA"}`.

I know that these values are not representative of how an actual span looks like, but if possible, we can replace them with something which is more human readable.